app/models/krikri/activity.rb

```diff
@@ -8,6 +8,15 @@ module Krikri
   # it is -- harvest, enrichment, etc.
   #
   class Activity < ActiveRecord::Base
+    # @!attribute agent
+    #    @return [Class] a Krikri::SoftwareAgent responsible for the activity
```

Comment:
When you say you are returning `[Class]` is this a Class Object or a String that represents a Class Object?